plugins/scaffolder/src/components/TaskPage/TaskErrors.tsx

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Box } from '@material-ui/core';
+import React from 'react';
+import { DismissableBanner } from '@backstage/core-components';
+import { TaskStream } from '../hooks/useEventStream';
+
+type TaskPageLinksProps = {
+  taskStream: TaskStream;
+};
+
+export const TaskErrors = ({ taskStream }: TaskPageLinksProps) => {
+  return taskStream.error ? (
+    <Box>
+      <DismissableBanner
+        id={String(Math.random())}
```

Comment:
This will update the id on every repaint, which probably is not desirable. The render phase should be side-effect-free. Maybe you could have something like this (untested written out of my own head):

```tsx
const id = useRef('');

useEffect(() => {
  id.current = String(Math.random());
}, [error]);
```

and then use `id={id.current}`

Reply:
This is much better, thanks!